pandas/core/groupby.py

```diff
@@ -929,6 +929,10 @@ def mean(self):
         Compute mean of groups, excluding missing values
 
         For multiple groupings, the result index will be a MultiIndex
+
+        Do not call this function via the numpy library. numpy's mean
```

Comment:
so see my note above. Let's leave these off for now, and just list them in another issue. These are all groupby/window functions